x/authority/keeper/policies.go

```diff
@@ -25,15 +29,25 @@ func (k Keeper) GetPolicies(ctx sdk.Context) (val types.Policies, found bool) {
 }
 
 // IsAuthorized checks if the address is authorized for the given policy type
-func (k Keeper) IsAuthorized(ctx sdk.Context, address string, policyType types.PolicyType) bool {
+func (k Keeper) IsAuthorized(ctx sdk.Context, msg sdk.Msg) (bool, error) {
```

Comment:
I don't think we need a error a any point
If policies is not found, no policies so not authorized
If more than 2 signers, not authorized

Reply:
The error adds some context to why the message is not authorized.
I considered logging the message but found using an error to be better for the user who is trying to broadcast .